server/core/src/miscServerFunct.cpp

```diff
@@ -2989,6 +2992,7 @@ irods::error setRECacheSaltFromEnv()
     return SUCCESS();
 } // setRECacheSaltFromEnv
 
+#if 0
 irods::error get_script_output_single_line(
     const std::string&              script_language,
     const std::string&              script_name,
```

Comment:
Don't forget about this. It's related to https://github.com/irods/irods/pull/7988/files#r1833170288.

Reply:
See https://github.com/irods/irods/pull/7988#discussion_r1835516816